grpc/src/from_server_proxy.rs

```diff
@@ -77,21 +77,40 @@ pub async fn forward_from_proto_to_ankaios(
                         )
                         .await?;
                 }
-                ExecutionRequestEnum::UpdateWorkloadState(obj) => {
+                FromServerEnum::UpdateWorkloadState(obj) => {
                     agent_tx
                         .update_workload_state(
                             obj.workload_states.into_iter().map(|x| x.into()).collect(),
                         )
                         .await?;
                 }
-                ExecutionRequestEnum::CompleteState(complete_state) => {
-                    agent_tx
-                        .complete_state(
-                            complete_state
-                                .try_into()
-                                .map_err(GrpcMiddlewareError::ConversionError)?,
-                        )
-                        .await?;
+                FromServerEnum::Response(response) => {
+                    // [impl->swdd~agent-adds-workload-prefix-id-control-interface-request~1]
+                    let request_id = response.request_id;
+
+                    match response
+                        .response_content
+                        .ok_or(GrpcMiddlewareError::ConversionError(format!(
+                            "Response content empty for response ID: {}",
```

Comment:
The trace message should surround variables with simple quotes: `'{}'`.

Reply:
fixed